### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by marcusturewicz** [marcusturewicz/jsonvalidate.net#39]
Lighthouse accessibility is 88%, remaining accessibility issues:
- Colour contrast; [default bootstrap colours lead to insufficient contrast](https://getbootstrap.com/docs/4.3/getting-started/accessibility/#color-contrast) - need to extend, possible via a [theme](https://getbootstrap.com/docs/4.3/getting-started/theming/).
- The page does not contain a heading, skip link, or landmark region

**Comment by marcusturewicz** [comment]
Closing as this is being handled in separate issues.
